internal/store/store.go

```diff
@@ -56,14 +60,23 @@ type BlockStore struct {
 	height int64
 }
 
+type BlockStoreOptions struct {
+	Metrics *Metrics
+}
+
 // NewBlockStore returns a new BlockStore with the given DB,
 // initialized to the last height that was committed to the DB.
-func NewBlockStore(db dbm.DB) *BlockStore {
+func NewBlockStore(db dbm.DB, o BlockStoreOptions) *BlockStore {
```

Comment:
Same. I'd propose adding a second constructor: `NewBlockStoreWithOptions`.

Reply:
Done as an optional parameter to the constructor of the blockstore.